### ISSUE TITLE
please document additions to MolarityScreenView

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/molarity#61]
Please document the additions to MolarityScreenView that were made in https://github.com/phetsims/molarity/commit/c6ec37eecc2ddaaa7593fad12aef8dd29a7e86cf, specifically `playAreaNode` and `controlAreaNode`.   

In general, additions and changes should have the same level of documentation as existing code. Thanks!

**Comment by zepumph** [comment]
@pixelzoom just so you know, most of the work is currently happening in the `screen-summary` branch ( #58). @twant I think this would be a great one for you to take on. IMO it would be best to do all development in the branch, including this additional documentation.

**Comment by twant** [comment]
Thanks for flagging this, @pixelzoom, and apologies for the sparse documentation! I'll take a look at this and add documentation to those portions (and any other new additions) on the screen-summary branch.

**Comment by twant** [comment]
@pixelzoom I've added some documentation to try to address this. Let me know what you think!

**Comment by pixelzoom** [comment]
👍 Looks great.  I made  one small doc tweak which you're welcome to review.  Closing.
